### DISCUSSION TITLE
Ignore cars

### DISCUSSION THREAD

**Discussion by JHCD** [flobz/psa_car_controller#238] [Ideas]
Hi,

Thank you for this great extension for my Corsa-E.

In addition to the electric vehicle, I have other vehicles in my account. This leads to tons of exceptions and error messages in the console/log and "unknown" displays in the UI. Is there a way to ignore vehicles? Deleting the entry in cars.json does not change anything.

Thanks for help!

Regards,
Jens

**Comment by flobz** [comment]
Please open an issue with log.
